android/beagle/src/main/java/br/com/zup/beagle/android/context/ContextDataManager.kt

```diff
@@ -30,21 +29,34 @@ import br.com.zup.beagle.android.utils.setContextBinding
 import br.com.zup.beagle.android.utils.setContextData
 
 internal class ContextDataManager(
-    private val jsonCreateTree: JsonCreateTree = JsonCreateTree(),
-    private val contextDataTreeHelper: ContextDataTreeHelper = ContextDataTreeHelper(),
-    private val contextPathResolver: ContextPathResolver = ContextPathResolver(),
-    private val contextDataEvaluation: ContextDataEvaluation = ContextDataEvaluation()
+    private val contextDataEvaluation: ContextDataEvaluation = ContextDataEvaluation(),
+    private val contextDataManipulator: ContextDataManipulator = ContextDataManipulator()
 ) {
 
+    private var globalContext: ContextBinding = ContextBinding(GlobalContext.getContext())
     private val contexts = mutableMapOf<Int, ContextBinding>()
     private val viewBinding = mutableMapOf<View, MutableSet<Binding<*>>>()
+    private val globalContextObserver: GlobalContextObserver = {
+        updateGlobalContext(it)
+    }
+
+    init {
+        contexts[Int.MAX_VALUE] = globalContext
```

Comment:
What do you think to change this constant to a variable to more readable something like 
```suggestion
           const val GLOBAL_CONTEXT_ID = Int.MAX_VALUE
               contexts[GLOBAL_CONTEXT_ID] = globalContext
```